app/services/schedule_service.rb

```diff
@@ -27,4 +29,21 @@ def create_timeline_entry(scheduling)
       details: scheduling,
     )
   end
+
+  def create_publish_intent(edition)
+    GdsApi.publishing_api.put_intent(edition.base_path, intent_payload)
+  end
+
+  def intent_payload
+    rendering_app = edition.document_type.publishing_metadata.rendering_app
+    publish_time = edition.scheduled_publishing_datetime
+    base_path = edition.base_path
+
+    {
+      publish_time: publish_time,
+      publishing_app: "content-publisher",
```

Comment:
Although I can't see this ever being a problem, could we use `PublishingApiPayload::PUBLISHING_APP` here? In fact, would this whole method fit better in that class?

Reply:
Good spot, `intent_payload` fits nicely in PublishingApiPayload!